test/refData/ref_parallel_df_codes.csv

```diff
@@ -0,0 +1,101 @@
+CD3(110:114)Dd,CD45(In115)Dd,CD4(Nd145)Dd,CD20(Sm147)Dd,CD33(Nd148)Dd,CD123(Eu151)Dd,CD14(Gd160)Dd,IgM(Yb171)Dd,HLA_DR(Yb174)Dd,CD7(Yb176)Dd
```

Comment:
Would it be possible to use a single ref_data set?

Reply:
I dont think so, the batch and the parallel methods generate different `df_codes` files